pages/index.tsx

```diff
@@ -1,26 +1,16 @@
 import type { NextPage } from "next";
 import Head from "next/head";
 import { Layout } from "../components/layout/Layout";
-import styles from "../styles/Home.module.css";
-
-import data from "../data/";
+import { Frontpage } from "../components/pages/Homepage";
 
 const Home: NextPage = () => {
   return (
     <Layout>
       <Head>
-        <title>Alman Akka</title>
+        <title>Tapahtumat - Alman Akka</title>
         <link rel="icon" href="/favicon.ico" />
```

Comment:
For some reason the browser reports an error not finding the favicon (even though it is visible). 
`GET http://localhost:3000/favicon.ico 404 (Not Found)`

Reply:
Hey that was a really good catch! The reason is, that the actual favicon is defined in the `pages/_document.tsx`, and as it uses an url, I removed the `favicon.ico` in the previous PR. So this line here is redundant, I will delete it.